src/manage_nsfs/manage_nsfs_events_utils.js

```diff
@@ -292,6 +292,26 @@ NoobaaEvent.BUCKET_ALREADY_EXISTS = Object.freeze({
     severity: 'ERROR',
     state: 'HEALTHY',
 });
+NoobaaEvent.BUCKET_OWNER_NOT_EXISTS = Object.freeze({
+    event_code: 'bucket_owner_not_exists',
+    message: 'Bucket owner does not exist',
+    description: 'The specified bucket owner does not exist in the system',
+    entity_type: 'NODE',
+    event_type: 'ERROR',
+    scope: 'NODE',
+    severity: 'ERROR',
+    state: 'DEGRADED'
```

Comment:
why state is DEGRADED?

Reply:
As per my understanding, by `DEGRADED` state we are referring that the operation can still be performed by making some adjustments to the command(could be some permission issue). Please correct me if I am wrong

Reply:
I think it affects the state of the cluster, @naveenpaul1 can you explain when event's state should be DEGRADED and when it should be HEALTHY?

Reply:
Yes @romayalon,  We have added DEGRADED stat to those events that could be the reason for potential I/O failers and core Noobaa config dir-related issue.

Reply:
@naveenpaul1 but does it do anything to the cluster? like ip movement etc? or it might happen only if health reported an error?

Reply:
@romayalon I also think this event should not have `DEGRADED` stat. For I/O events we added DEGRADED stat assuming could be because of some internal issue something related to cluster or storage itself.

Reply:
Thanks, I will update it to HEALTHY then.